lib/gui/app.js

```diff
@@ -162,7 +175,9 @@ app.run(($timeout, DriveScannerService, DrivesModel, ErrorService) => {
     // available drives list has changed, and incorrectly
     // keeps asking the user to "Connect a drive".
     $timeout(() => {
-      DrivesModel.setDrives(drives);
+      if (!_.isEqual(DrivesModel.getDrives(), drives)) {
+        DrivesModel.setDrives(drives);
```

Comment:
The `setDrives()` function should already be preventing the same drives from getting set again. Is this not the case? If not, can we fix it on `DrivesModel`?